### PR TITLE
Add CANCELED status when wait_for_job

### PR DESCRIPTION
**Pull request by Mrart**
Add CANCELED status when wait_for_job #194

### CODE REVIEW

File: controllers/flinkcluster_submit_job_script.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ function wait_for_job() {
 	while true; do
 		echo -e "\nWaiting for job to finish..."
 		list_jobs
-		if list_jobs | grep -e "(FINISHED)" -e "(FAILED)"; then
+		if list_jobs | grep -e "(FINISHED)" -e "(FAILED)" -e "(CANCELLED)"; then
```
Comment (functicons) [inline controllers/flinkcluster_submit_job_script.go]: I tested, seems it should be `CANCELED` instead of `CANCELLED`.

Comment (Mrart) [reply controllers/flinkcluster_submit_job_script.go]: yes, It's my fault

**Pull request merged**